lib/linguist/languages.yml

```diff
@@ -1758,6 +1758,16 @@ Groovy Server Pages:
   codemirror_mode: htmlembedded
   codemirror_mime_type: application/x-jsp
   language_id: 143
+HAProxy:
+  type: data
+  extensions:
+  - ".cfg"
+  filenames:
+  - hapee-lb.cfg
```

Comment:
I found [only 2 such files](https://github.com/search?utf8=%E2%9C%93&q=filename%3Ahapee-lb.cfg&type=Code) on GitHub.com. That won't be enough to warrant support in Linguist.

Reply:
That makes sense. The Enterprise version calls it hapee-lb.cfg, but id expect way more people to be hosting the Community Edition haproxy.cfg. Should I remove the former?

Reply:
> Should I remove the former?

Yes :)

Reply:
Removed it